plugin/trino-delta-lake/pom.xml

```diff
@@ -416,6 +416,8 @@
                                 <exclude>**/TestDeltaLakeTableWithCustomLocationUsingGlueMetastore.java</exclude>
                                 <exclude>**/TestDeltaLakeRenameToWithGlueMetastore.java</exclude>
                                 <exclude>**/TestDeltaLakeGcsConnectorSmokeTest.java</exclude>
+                                <exclude>**/TestDeltaLakeRegisterTableProcedureUsingGlueMetastore.java</exclude>
+                                <exclude>**/TestDeltaLakeWithExistingLocationUsingGlueMetastore.java</exclude>
```

Comment:
I can't find `TestDeltaLakeWithExistingLocationUsingGlueMetastore` in this PR.

Reply:
I missed updating in `pom.xml`. Now, this class is not needed. So removed it from here.